todos.go

```diff
@@ -0,0 +1,109 @@
+package gitlab
+
+import "time"
+import "fmt"
+
+// TodosService handles communication with the todos related methods of
+// the Gitlab API.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/8-16-stable/doc/api/todos.md
+type TodosService struct {
+	client *Client
+}
+
+// Todo represents a GitLab todo.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/8-16-stable/doc/api/todos.md
+type Todo struct {
+	ID      int `json:"id"`
+	Project struct {
+		ID                int    `json:"id"`
+		HTTPURLToRepo     string `json:"http_url_to_repo"`
+		WebURL            string `json:"web_url"`
```

Comment:
correct, the example response from the [docs](https://docs.gitlab.com/ce/api/todos.html#mark-a-todo-as-done) does not have them however, if I curl the v4 resource, I can see them listed.  

curl --header "PRIVATE-TOKEN: $PRIVATE_TOKEN" https://gitlab.com/api/v4/todos 

"project": {
      "id": 3048683,
      "http_url_to_repo": "https://gitlab.com/eguevara/helloworld.git",
      "web_url": "https://gitlab.com/eguevara/helloworld",
      "name": "helloworld",
      "name_with_namespace": "Erick Guevara / helloworld",
      "path": "helloworld",
      "path_with_namespace": "eguevara/helloworld"
    },